ci.sh

```diff
@@ -97,6 +97,11 @@ cargo build \
     --target thumbv6m-none-eabi \
     --profile ci
 
+cargo build \
+    --manifest-path examples/i2c-passthru/rp2350/Cargo.toml \
+    --target thumbv8m.main-none-eabihf \
```

Comment:
Please add this target to the setup steps around https://github.com/OneVariable/poststation-util/pull/19/files#diff-95bb4f36f7ea53c8ed1af485b779550e22e21e4686c5f126a64eb6b074a8f99dL8-L11